lib/gui/server.js

```diff
@@ -64,6 +64,11 @@ exports.start = ({paths, hermione, guiApi, configs}) => {
         }
     });
 
+    server.post('/stop', (req, res) => {
+        res.sendStatus(200);
+        process.exit(0);
```

Comment:
here we should stop also hermione process, otherwise the tests will still be executed

Reply:
It should be stopped by `onExit` callback below, see [`signal-exit`](https://github.com/tapjs/signal-exit) docs:

> explicitly having `process.exit(code)` called.

Reply:
I wanted to say that now there is a problem, that before completing the hermione process we should close browsers. Otherwise browsers are hanging. We will add api for this to hermione soon and then merge your PR.

You can reproduce it when run test with long timeout (30 sec for example) on local selenium server.

Reply:
> We will add api for this to hermione soon

Hi, can I track the progress of this task anywhere?

Reply:
> Hi, can I track the progress of this task anywhere?

Hello. Unfortunately no one from our team is doing this task now.

Reply:
I think I found a way to do it. Can you please check again?